webapp-django/crashstats/base/apps.py

```diff
@@ -0,0 +1,42 @@
+from django.apps import AppConfig
+from django.conf import settings
+
+import markus
+
+
+class BaseConfig(AppConfig):
+    name = 'crashstats.base'
+
+    def ready(self):
+        if settings.LOCAL_DEV_ENV:
+            # If we're in the local development environment, then use the
+            # logging and statsd backends
+            backends = [
+                {
+                    'class': 'markus.backends.logging.LoggingMetrics',
+                },
+                {
+                    'class': 'markus.backends.statsd.StatsdMetrics',
+                    'options': {
+                        'statsd_host': settings.STATSD_HOST,
+                        'statsd_port': settings.STATSD_PORT,
+                        'statsd_prefix': settings.STATSD_PREFIX,
+                    }
```

Comment:
Datadog extends statsd adding tags and some other things. That doesn't work with statsd at all. Markus supports both, though, so we use the statsd backend in the local dev environment and then you can see some things in Graphite in addition to the logs.